src/screens/scanning/ManualVoucherScreen.tsx

```diff
@@ -116,13 +116,21 @@ export default function ManualVoucherScreen({
         <FormContainer>
           <FieldContainer>
             <InputTitleText>Serial Number</InputTitleText>
-            <InputField
-              onChange={onChangeSerialNumber}
-              value={serialNumber}
-              placeholder="Enter Number"
+            <OTPTextInput
+              inputCount={7}
+              tintColor={Colors.magenta}
+              defaultValue={serialNumber}
+              handleTextChange={onChangeSerialNumber}
+              containerStyle={{ marginTop: 10 }}
+              textInputStyle={{
+                borderWidth: 1,
+                borderRadius: 2,
+                width: 30,
+              }}
               isValid={!showInvalidError}
-              keyboardType="number-pad"
-            />
+            >
+              {}
```

Comment:
not sure if there's a special reason we need this on IOS, but on android it doesn't seem to be any different than just have <OTPTextInput options />?

Reply:
good catch